example/main.dart

```diff
@@ -38,6 +40,91 @@ Future<Null> main(List<String> rawArgs) async {
   }
 }
 
+Future<Null> captureCompleteExampleEvent(SentryClient client) async {
```

Comment:
Maybe leave a doc comment explaining what this example is trying to demonstrate?